cmd/mattermost/commands/server.go

```diff
@@ -78,6 +80,11 @@ func runServer(configStore config.Store, disableConfigWatch bool, usedPlatform b
 		return serverErr
 	}
 
+	// Start server so prometheus scrapers can fetch metric collected by prometheus clients
+	go startTelemetryServer(
+		server.Config().ExperimentalSettings.TelemetryServerListenAddress,
+		server.Log.StdLog(mlog.String("source", "telemetry_server")))
```

Comment:
There is no use of passing this logger instance from here. It can be created & used in the `startTelemetryServer` function itself. Ideally logger should be initiated only once, see if we can do that, other wise atleast remove this parater which can be directly defined & used in below function itself.

Reply:
I want to use the same logger which is getting created in server.